authenticity_test.cpp

```diff
@@ -416,7 +393,7 @@ void PasskeyTest::TestPasskey() const
     BOOST_TEST(!fs::exists("passkey"));
     Authenticity::ResetCache();
     BOOST_TEST_EQUAL
-        ("Unable to read passkey file 'passkey'. Try reinstalling."
+        ("Unable to read passkey file './passkey'. Try reinstalling."
```

Comment:
This looks like a problem. I.e. the previous message seems better, can we avoid the leading `./` here (and below, of course)?